pages/home.js

```diff
@@ -1,10 +1,12 @@
 import { useState } from "react";
 import Head from "next/head";
 import Button from "../components/button";
-import styles from "../styles/home.module.css";
 import DashNavbar from "../components/dashNavbar";
 import { isAuthenticated, redirectHome } from "../lib/serverSideAuth";
+import HeirloomSettingsModal from "../components/heirloomSettingsModal";
```

Comment:
[nit] I forgot to mention this and please disregard but one good practice is to always include the imports in alphabetical order. Again this is a nit, so don't worry about it too much

Reply:
No this is a good point. I actually thing there's an eslint config that fixes this for us